src/ISDataMappings.cpp

```diff
@@ -1064,6 +1066,28 @@ static void PopulateFlashConfigMappings(map_name_to_info_t mappings[DID_COUNT])
     ASSERT_SIZE(totalSize);
 }
 
+/**
+ * Maps DID_EVENT for SDK
+*/
+static void PopulateISEventMappings(map_name_to_info_t mappings[DID_COUNT])
+{
+     typedef did_event_t MAP_TYPE;
+    map_name_to_info_t& m = mappings[DID_EVENT];
+    uint32_t totalSize = 0;
+
+
+    ADD_MAP(m, totalSize, "Time stamp of message", timeMs, 0, DataTypeUInt32, uint32_t, 0);
+    ADD_MAP(m, totalSize, "Senders serial number", senderSN, 0, DataTypeUInt32, uint32_t, 0);
+    ADD_MAP(m, totalSize, "Sender hardware type", senderHdwType, 0, DataTypeUInt16, uint16_t, 0);
+
+    ADD_MAP(m, totalSize, "protocol", protocol, 0, DataTypeUInt16, uint16_t, 0);
```

Comment:
"protocol" (and length, data below) are Human-readable, they should have proper grammar/capitalization.